content/pt-br/docs/concepts/cluster-administration/flow-control.md

```diff
@@ -0,0 +1,630 @@
+---
+title: Prioridade e imparcialidade da API
+content_type: concept
+min-kubernetes-server-version: v1.18
+---
+
+<!-- overview -->
+
+{{< feature-state state="beta"  for_k8s_version="v1.20" >}}
+
+Controlar o comportamento do servidor da API Kubernetes em uma situação de sobrecarga
+é uma tarefa chave para administradores de cluster. O {{< glossary_tooltip
+term_id="kube-apiserver" text="kube-apiserver" >}} tem alguns controles disponíveis
+(ou seja, as _flags_ `--max-requests-inflight` e `--max-mutating-requests-inflight`) 
+para limitar a quantidade de trabalho pendente que será aceito, 
+evitando que uma grande quantidade de solicitações de entrada sobrecarreguem, e
+potencialmente travando o servidor da API, mas essas _flags_ não são suficientes para garantir
+que as solicitações mais importantes cheguem em um período de alto tráfego.
+
+O recurso de prioridade e imparcialidade da API (APF) é uma alternativa que melhora
+as limitações mencionadas acima. A APF classifica
+e isola os pedidos de uma forma mais refinada. Também introduz
+uma quantidade limitada de filas, para que nenhuma solicitação seja rejeitada nos casos
+de sobrecargas muito breves. As solicitações são despachadas das filas usando uma
+técnica de filas justa para que, por exemplo, um
+{{< glossary_tooltip text="controller" term_id="controller" >}} não precise
+negar as outras requisições (mesmo no mesmo nível de prioridade).
+
+Esse recurso foi projetado para funcionar bem com controladores padrão, que
+usam informantes e reagem a falhas de solicitações da API com exponencial
+back-off, e outros clientes que também funcionam desta forma.
+
+{{< caution >}}
+Solicitações classificadas como "de longa duração" — principalmente relógios — não são
```

Comment:
`principalmente relógios` I did get this sentence, what do we mean by `relógios?`